Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -377,8 +378,29 @@ void HexEditorWidget::update_inspector_values(size_t position)
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::UTF16, "");
     }
 
+    auto selected_bytes = m_editor->get_selected_bytes();
+
+    StringBuilder ascii_string_builder;
+    for (size_t i = 0; i < selected_bytes.size(); i++) {
+        char byte_char = static_cast<char>(selected_bytes[i]);
+        ascii_string_builder.append(byte_char);
+    }
+
+    value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCIIString, ascii_string_builder.to_deprecated_string());
```

Comment:
This StringBuilder isn't needed, you can instead treat `selected_bytes`'s data as a DeprecatedString directly:

```suggestion
    value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::ASCIIString, DeprecatedString { selected_bytes });
```